ResumableUpload.CS.Sample/ResumeableUploadSample.cs

```diff
@@ -0,0 +1,241 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Google.Apis.Http;
+using Google.Apis.Requests;
+using Google.Apis.Services;
+using Google.Apis.Util;
+using Google.Apis.YouTube.v3;
+using Google.Apis.YouTube.v3.Data;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Upload;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace ResumableUpload.CS.Sample
+{
+    class ResumeableUploadSample
+    {
+        /// <summary>
+        /// A C# .NET sample for the YouTube Data API using ResumableUpload.
+        /// 
+        /// See https://developers.google.com/api-client-library/dotnet/guide/media_upload for more details regarding ResumableUpload.
+        /// See https://developers.google.com/resources/api-libraries/documentation/youtube/v3/csharp/latest/ for YouTube Data API documentation.
+        /// See https://developers.google.com/youtube/v3/getting-started for details of the underlying API calls used by the .NET Client Library.
+        /// See https://github.com/google/google-api-dotnet-client/releases for .NET Client Library Release Notes.
+        /// See https://www.nuget.org/packages/Google.Apis.YouTube.v3/ for information on the .NET Client Library NuGet package.
+        /// 
+        /// Prepare for running the sample program:
+        ///   1. Add Reference to System.Net.Http
+        ///   2. In NuGet Package Manager Console, enter:
+        ///       Install-Package Google.Apis.YouTube.v3 
+        ///   3. Click Tools / NuGet Package Manager / Manage NuGet Packages for Solution
+        ///   4. Click on Updates / nuget.org and install all update packages
+        ///   5. Insert your ClientId and ClientSecret in the source code below.
+        ///   6. Change strFilePath below to specify the test video file to be uploaded.
+        /// </summary>
+        const int MAX_RESUME_RETRIES = 3;
+        static VideosResource.InsertMediaUpload objVideosInsertRequest = null;
+        static System.Threading.CancellationToken objCancellationToken;
+
+        static void Main(string[] args)
+        {
+            ExecuteUploadProcess().Wait();
+        }
+
+        /// <summary>
+        /// Execute the upload of one file to YouTube.
+        /// </summary>
+        private static async Task ExecuteUploadProcess()
+        {
+            const int intMinimumChunkSize = 256 * 1024; // .25MB (256K)
```

Comment:
Most C# style guides ban the use of hungarian notation. Mind renaming these? In Visual Studio I believe CTRL+R CTRL+R is the shortcut for renaming a variable.